locale/en/hammer_cli_foreman_discovery.po

```diff
@@ -0,0 +1,99 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: hammer_cli_foreman_discovery 0.0.1\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-01-21 14:11+0200\n"
+"PO-Revision-Date: 2015-01-21 12:06+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```

Comment:
when .tx/config looked like this:
```
[foreman.hammer-cli-foreman]
file_filter = locale/<lang>/hammer-cli-foreman-discovery.po
```
which is what was in hammer bootdisk I got the strings for hammer-cli-foreman.
I changed it to this:
```
[foreman.hammer-cli-foreman-discovery]
file_filter = locale/<lang>/hammer-cli-foreman-discovery.po
```
and then no strings where translated.